lib/opensusebasetest.pm

```diff
@@ -595,7 +595,7 @@ sub wait_grub {
     elsif (match_has_tag('encrypted-disk-password-prompt')) {
         # unlock encrypted disk before grub
         workaround_type_encrypted_passphrase;
-        assert_screen "grub2", 15;
+        assert_screen "grub2", 30;
```

Comment:
hm, I am not sure why there was explicit selection for 15 here. 30 is default so you can just as well leave it out. It might be that there was explicit selection for a lower value so that the chance is higher one would fail still in the grub menu on a needle mismatch and a timeout hasn't run out. Does it *really* take more than 15s to unlock the password on grub startup and then show the grub menu?

If you keep this change out I would have an easier time with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9035 which I hope you appreciate as well :)

Reply:
This is just WIP experiment, because of slow aarch64  https://openqa.suse.de/tests/3644735. I need to look on this part more.

Reply:
With change to 30, aarch64 booted https://openqa.suse.de/tests/3644752.

Reply:
This part was removed.